insilicho/run.py

```diff
@@ -207,12 +207,12 @@ def flex2_sampling(
     # sample across small time range, add noise
     for k, var in res_map.items():
         var = np.maximum(var, parameters.EPSILON)
-        if k == "time":
+        if k in ["time", "V"]:
             res[k] = var[idx].tolist()
         else:
             res[k] = np.maximum(
-                np.random.normal(
-                    loc=var[idx], scale=float(sampling_stddev), size=len(idx)
+                var[idx] * np.random.normal(
+                    loc=1.0, scale=float(sampling_stddev), size=len(idx)
```

Comment:
1. Just noting that L64 also tries to implement a similar outcome, i.e. to get values that are scaled (i.e. relative_std_dev) so maybe good to standardize the logic? 

2. I would rename args and docstring with prefix "rel" or "relative" ala L53 to make it clear the noise is relative.

Reply:
makes sense -- I could abstract the logic out to be applied to both parameters and flex2 measurements. good point about arg and docstring